### PR TITLE
Script/rails

### PR DESCRIPTION
**Pull request by ncanderson**
works and install ruby 2.3.0 and rails 4.2.5. 

still trying to figure out how to pass the arguments into the script so the user can specify the versions.

### CODE REVIEW

**Comment by davidmyersdev** [comment]
Hey there! I just reviewed this. You will need to do a couple of things before I can merge. First of all, you will need to create a new PR on the drmyersii:develop branch (I just created this for merging in PRs). Also, make sure that you have pulled in any updates from origin (my copy). If you aren't up to date with my changes, GitHub won't allow me to merge the PR in. Thanks!

**Comment by ncanderson** [comment]
Hi David,
Revisiting the scripts and problems I was trying to explain to you at pair
programming. A things I wanted to run by you before creating a new pull
request.

  tl;dr: Using a ppa to install ruby causes a lot of trouble trying to
install ruby gems, because of file permissions. Using rbenv to install ruby
and rails adds time to vagrant up but is among the commonly used and
accepted methods of installing ruby/rails.

  Having a lot of troubling with Ruby from the ppa. The basic package
installs well (as you saw), but there are a lot of complications when it
comes to using gems (ruby libraries, which includes Rails). Mostly it's
related to the permissions generated during vagrant up. Pre-fab ruby/rails
installers created a directory and modify the PATH such that all ruby/rails
versions and associated gems have their own directory under the user. The
ppa, which installs via apt-get, by default installs above the user
directory, putting the gems in various folders (/var/lib and /usr/local).
Installing gems via sudo is not recommended in the ruby community. Trying
to predict where every gem might end up and changing those permissions
seems both tedious and fraught, as I don't know enough to know which
files/directories need to be owned by root. Finally, a brief look into
modifying where apt-get installs the ppa seems like it's beyond me.

  I guess what I'm getting to is that using a ppa for ruby and ruby on
rails is getting into some pretty advanced (for me) stuff. I'd like to
modify the scripts for ruby and ruby on rails to use rbenv, which extends
the vagrant up time (by a lot) but will eliminate most of the headaches.

  Also curious what this error means: dpkg-preconfigure: unable to re-open
stdin: No such file or directory

thanks,
nate

**Comment by davidmyersdev** [comment]
I will need to take a look at your current setup to get an idea of what might be going on. Which branch/commit should I take a look at in your forked project? Is it currently functional? If not, what will you need to do to get it functional? Thanks for all the hard work on this!

**Comment by davidmyersdev** [comment]
Closing this since a new PR (#22) is open.
